### PR TITLE
[SHOT] v1.1 and build on experimental platforms

### PR DESCRIPTION
**Pull request by odow**

### CODE REVIEW

**Comment by giordano** [comment]
You probably don't need to update the SDK for aarch64-apple-darwin

File: C/Coin-OR/SHOT/build_tarballs.jl [hunk 1]
```diff
@@ -20,7 +15,7 @@ cd $WORKSPACE/srcdir/SHOT
 git submodule update --init --recursive
 # Disable run_source_test in CppAD
 atomic_patch -p1 ../patches/CppAD.patch
-if [[ "${target}" == *-darwin* ]]; then
+if [[ "${target}" == *-x86_64-apple-darwin* ]]; then
```
Comment (giordano) [inline C/Coin-OR/SHOT/build_tarballs.jl:18]:
```suggestion
if [[ "${target}" == x86_64-apple-darwin* ]]; then
```

**Comment by odow** [comment]
I'm not that sad if we can't build SHOT on the M1: https://github.com/phracker/MacOSX-SDKs/issues/41

I'm surprised we made it this far through the COIN-OR stack before finding a problem!

**Comment by giordano** [comment]
But we use the 11.1 SDK on that platform:
https://github.com/JuliaPackaging/Yggdrasil/blob/0173b33d73d6580c833f84f972a1fc25d2b1ed25/0_RootFS/gcc_common.jl#L240
I don't understand the error, there may be something else wrong going on.

**Comment by odow** [comment]
Thanks @giordano! 20 PRs later and we made it through the COIN-OR stack with very few issues, which I was not expecting...

**Comment by giordano** [comment]
For the record, we are getting the wrong minimum macOS version from the wrappers for `aarch64-apple-darwin`, that's why this was failing here.  This is (hopefully) going to be fixed by https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/193.

> Thanks @giordano! 20 PRs later and we made it through the COIN-OR stack with very few issues, which I was not expecting...

Maybe we have a robust toolchain at this point :stuck_out_tongue_winking_eye:

**Pull request merged**